tutorials/dotnet-mvms-quickstart.md

```diff
@@ -201,7 +199,7 @@ Learn how to use App Engine with other Google Cloud Platform products:
 **Learn to use Cloud SQL.** Cloud SQL is a fully-managed, relational MySQL database for your applications. </walkthrough-tutorial-card><walkthrough-alt>Learn more in the [Cloud SQL documentation](https://cloud.google.com/appengine/docs/flexible/dotnet/using-cloud-sql).</walkthrough-alt>
 
 <walkthrough-tutorial-card url=https://cloud.google.com/appengine/docs/flexible/dotnet/using-cloud-storage icon="STORAGE_SECTION" label="cloudStorage">
-**Learn to use Cloud Storage** Cloud Storage is a powerful and simple object
+**Learn to use Cloud Storage.** Cloud Storage is a powerful and simple object
```

Comment:
Recommend avoiding language like powerful, simple, quick, easy, etc.

Reply:
Good point. I'll pass this along.